recipe/meta.yaml

```diff
@@ -1,49 +1,51 @@
-{% set version = "16.7.5" %}
+{% set version = "20.0.20" %}
 
 package:
   name: virtualenv
   version: {{ version }}
 
 source:
-  url: https://github.com/pypa/virtualenv/archive/{{ version }}.tar.gz
-  sha256: b14f2548f9d9b3db3309756069d2df9cbb8eb1cbd5f968437fc4580a6dbf19b6
-  patches:
-    - 0001-Mach-O-Copy-any-dynamic-libraries-and-symlinks-to-th.patch
-    - 0002-ELF-Copy-any-dynamic-libraries-and-symlinks-to-the-c.patch
```

Comment:
Do we recall why these patches were dropped? These were needed to workaround some issues on macOS and Linux regarding library loading (particularly `libpython`). Curious if we know whether upstream fixed these issues and if so where those fixes were included

Reply:
Interesting thanks for the info (and the work) :)

Ok good to know. Saw a few issues/PRs that looked potentially related, but couldn't tell which were relevant. As no one has said anything about the Conda package having issues, guessing the original issues are indeed fixed (which is great news!)